src/reservation.js

```diff
@@ -0,0 +1,44 @@
+const reservation = (movie) => `
+<div class="modal modal-reservation" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
```

Comment:
- [ ] **REQUIRED** The reservation should throw an error message to prompt the user to select the second date. Please consider implementing that fix